lucien-ios/Models/User.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  User.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 9/29/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+import UIKit
+
+struct User: Decodable {
+    var firstName: String
+    var lastName: String
+    var email: String
+    var googlePictureURL: String
+    var userID: String
+    var googleUserID: String
+    var createdAt: Date
+    var updatedAt: Date
+
+    enum CodingKeys: String, CodingKey {
+        case user
+        case token
+    }
+    enum UserKeys: String, CodingKey {
+        case firstName = "first_name"
+        case lastName = "last_name"
+        case email = "email"
+        case googlePictureURL = "google_picture_url"
+        case userID = "id"
+        case googleUserID = "google_user_id"
+        case createdAt = "created_at"
+        case updatedAt = "updated_at"
+    }
+    init(from decoder: Decoder) throws {
+        let values = try decoder.container(keyedBy: CodingKeys.self)
+        let token = try values.decode(String.self, forKey: .token)
```

Comment:
Is this crashing for `getCurrentUser` right now? Or rather are you calling `getCurrentUser` as of this PR, or only in the dashboard branch?

Reply:
I'm only calling it in the dashboard branch and it's not crashing; it's just grabbing the name of the current user if that person is already signed in. If asked to sign in via google the name shows up, otherwise if taken straight to the dashboard the current user returns No first name and No last name. 
<img width="715" alt="screen shot 2017-10-06 at 12 17 41 pm" src="https://user-images.githubusercontent.com/6693598/31298904-365909c8-aaba-11e7-8c7f-04d3dd1839ae.png">